.github/workflows/changelogs.yml

```diff
@@ -33,6 +33,14 @@ jobs:
         with:
           path: ~/.cabal/store
           key: linux-store-changelogs
+      # See https://github.com/haskell/cabal/pull/8739
+      - name: Sudo chmod to permit ghcup to update its cache
+        run: |
+          if [[ "${{ runner.os }}" == "Linux" ]]; then
```

Comment:
Just a little smartass remark:  This conditional can reside at the level of the action language, like:
```
      if: ${{ runner.os == 'Linux' }}
```
This will then display the step as skipped on macOS/Windows.  If one wants such a thing.
(This smartass remark is part of my personal quest to find best practices for github workflows. No need to act.)

Reply:
Smartass indeed. :D